src/components/qa/QuestionsAnswers.jsx

```diff
@@ -0,0 +1,19 @@
+import React, {useState, useEffect} from 'react';
+import QuestionList from './QuestionList.jsx';
+import Search from './Search.jsx';
+
+const QuestionsAnswers = ()  => (
+  <section>
+  <div id='question-answers'>
```

Comment:
forget to indent, but it's fine, you could fix it later

Reply:
Good catch